### DISCUSSION TITLE
video stream

### DISCUSSION THREAD

**Discussion by rudyhadoux** [ionic-team/capacitor#3966] [General]
Hi,
Is there a way to get a video stream (MediaStream) from Capacitor Camera.getPhoto( ) ?

**Comment by jcesarmobile** [comment]
No, camera plugin launches an intent to the camera app on android and uses the native image picker on iOS, none of those allow to get the stream.

On android you can use web technologies to get the stream with getUserMedia and put it in a video tag, but sadly that’s not available on iOS apps.

**Reply by fosteman** [reply]
Well, is there a way to get a stream from camera for post-processing on IOS at all @jcesarmobile ?

**Comment by rudyhadoux** [comment]
Ok thanks. 
According to that 
https://stackoverflow.com/questions/64049506/issue-with-webrtc-getusermedia-in-ios-14-safari-and-phone-sleep-unlock
iOS 12 and 13 accept getUserMedia.

**Reply by jcesarmobile** [reply]
Yeah, but only on Safari, not on the `WKWebView` that Capacitor uses.

On iOS 14.3 it also works on `WKWebView`, but only when loading an external site that uses https to serve, not possible from local assets.

**Comment by rudyhadoux** [comment]
Hope that the Apple team will evolve ...
